global-script.lua

```diff
@@ -2462,7 +2470,9 @@ function BoardCallback(obj,pos,rot,extra)
     obj.interactable = false
     obj.setLock(true)
     obj.setRotationSmooth(rot, false, true)
-    obj.setPositionSmooth(pos, false, true)
+    local scaleFactor = scaleFactors[SetupChecker.getVar("scaleBoard")]
+    obj.setPositionSmooth(posOrigin+pos*scaleFactor.position, false, true)
+    obj.scale(scaleFactor.size)
```

Comment:
presumably there is no setScaleSmooth equivalent is there?

Reply:
There does not appear to be.